src/components/Searchbar/Search.js

```diff
@@ -17,14 +18,27 @@ export default function SearchBar({
   const handleClick = () => {
     refInput.current.focus();
   };
-  //----- if input -> setResult with filterd base array-----//
-  function filteredProduct(input) {
-    let inputReg = new RegExp(input, 'i');
+  //-----Language is EN - if input  -> setResult with filterd base array-----//
+  function filteredProductEN(input) {
+    let inputRegEN = new RegExp(input, 'i');
```

Comment:
Why use `let` instead of `const`?

Reply:
At first i tried to combine EN & DE. i got a console log error which referred to the variable. Didn't really know how to handle that and changed to let - i've also seen this in my little tutorial on regular expressions. Now i changed both to const and it works. Thanks. Im not fully understanding regExp at the moment and it took nearly a whole day for this function to work.